libs/trades/src/lib/trades-table.tsx

```diff
@@ -0,0 +1,88 @@
+import type { AgGridReact } from 'ag-grid-react';
+import { AgGridColumn } from 'ag-grid-react';
+import { forwardRef, useMemo } from 'react';
+import { AgGridDynamic as AgGrid } from '@vegaprotocol/ui-toolkit';
+import type { TradeFields } from './__generated__/TradeFields';
+import {
+  formatNumber,
+  getDateTimeFormat,
+  t,
+} from '@vegaprotocol/react-helpers';
+import type { CellClassParams, ValueFormatterParams } from 'ag-grid-community';
+import BigNumber from 'bignumber.js';
+import { sortTrades } from './trades-data-provider';
+
+export const UP_CLASS = 'text-vega-green';
+export const DOWN_CLASS = 'text-vega-pink';
+
+const changeCellClass =
+  (dataKey: string) =>
+  ({ api, value, node }: CellClassParams) => {
+    const rowIndex = node?.rowIndex;
+    let colorClass = '';
+
+    if (typeof rowIndex === 'number') {
+      const prevRowNode = api.getModel().getRow(rowIndex + 1);
+      const prevValue = prevRowNode?.data[dataKey];
+      const valueNum = new BigNumber(value);
+
+      if (valueNum.isGreaterThan(prevValue)) {
+        colorClass = UP_CLASS;
+      } else if (valueNum.isLessThan(prevValue)) {
```

Comment:
Is it possible that `valueNum` could ever be equal to `prevValue`?

Reply:
It is! and in that case we don't apply a class so it just inherits the default text color (up, green, down red, no change white)